packages/@tailwindcss-postcss/src/index.ts

```diff
@@ -263,6 +274,7 @@ function tailwindcss(opts: PluginOptions = {}): AcceptedPlugin {
               DEBUG && I.end('AST -> CSS')
 
               DEBUG && I.start('Lightning CSS')
+              // @ts-ignore
```

Comment:
Can we drop this now or nah?

Reply:
Pretty sure this is a leftover from a wrong build, my bad!